Orso.Arpa.Infrastructure/Authentication/JwtGenerator.cs

```diff
@@ -119,5 +75,25 @@ private RefreshToken GernerateRefreshToken(User user, string ipAddress)
                 now
             );
         }
+
+        public async Task<ClaimsIdentity> GetClaimsIdentity(User user)
```

Comment:
this is not a "Get" Method, it rather Creates the claims identity for a given user